src/renderer/src/scenes/main/MainScene.tsx

```diff
@@ -59,7 +59,13 @@ const NavItem: Component<NavItemProps> = ({ children, value, icon }) => {
     <button
       class="nav-item"
       data-selected={mainActiveTab() === value}
```

Comment:
Variable `value` is not good name in my opinion. I would like to see something along the lines of `selectedTab`